typo.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Krozaro</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./src/krozaro.styles.min.css" />
+    <link rel="stylesheet" href="./styles.css" />
+    <link href="prism/prism.css" rel="stylesheet" />
+    <link rel="icon" href="./src/img/favicon.ico" type="image/icon">
+  </head>
+
+  <body class="menuGrid">
+<aside class="sidebar-container">
+    
+    <a href="./index.html" class="docs-logo">
+        <img src="./src/img/docs-logo.png"/>
+    </a>
+
+    <div class="sidebar">
+      <li class="list-head">Getting Started</li> 
+      <a href="./docs.html"><li>Introduction</li></a>
+       <a href="./installation.html"> <li>Installation</li> </a>
+       <a href="./typo.html"> <li class="active">Typography</li> </a>
+       <a href="./theme.html"> <li>Theme</li> </a>
+      
+      <li class="list-head">UI Components</li>      
+       <a href="./src/components/alert/alert.html"> <li>Alert</li> </a>
+       <a href="./src/components/avatar/avatar.html"> <li>Avatar</li> </a>
+       <a href="./src/components/badge/badge.html"> <li>Badge</li> </a>
+       <a href="./src/components/button/button.html"> <li>Button</li> </a>
+       <a href="./src/components/card/card.html"> <li>Card</li> </a>
+       <a href="./src/components/grid/grid.html"> <li>Grid</li> </a>
+       <a href="./src/components/image/image.html"> <li>Image</li> </a>
+       <a href="./src/components/input/input.html"> <li>Input</li> </a>
+       <a href="./src/components/lists/lists.html"> <li>Lists</li> </a>
+       <a href="./src/components/modal/modal.html"> <li>Modal</li> </a>
+       <a href="./src/components/navigation/navigation.html"> <li>Navigation</li> </a>
+       <a href="./src/components/rating/rating.html"> <li>Rating</li> </a>
+       <a href="./src/components/toast/toast.html"> <li>Toast</li> </a>
+      </div>
+</aside>
+<main class="doc-main"> 
+    <nav>
+        <ul class="nav-group start">
+          <li class="nav-active"> <a href="./docs.html">Getting Started</a></li>
+            <li><a href="./src/components/alert/alert.html">UI Components</a></li>
+        </ul>
+        <ul class="nav-group end">
+        <li><a href="https://github.com/shangkaul/krozaro_component_lib"><i class="icon icon-md fa fa-github"></i></a></li>
+        <li></li>
+    </ul>
+    
+    </nav>
+    <div class="doc-content">
+        <section class="component">
+            <h1>Typography</h1>
+          <p class="md">The Krozaro library provides several flavours of text content that can adapt to your ever changing product needs. Examples of different variations of textual content:</p>  
+          </section>
+<section class="component">
+          <h2>Heading</h2>
+          <section class="demo">
+          
+            <h1> Heading 1</h1>
+            <h2> Heading 2</h2>
+            <h3> Heading 3</h3>
+            <h4> Heading 4</h4>
+            <h5> Heading 5</h5>
+
+          </section>
+          <pre>
```

Comment:
You have given custom line height for each heading, instead you can remove line height for this example and use flexbox on container div for better alignment and presentation.
It will look better.


![Typography](https://user-images.githubusercontent.com/90456667/153391953-c76f1b66-22c9-4c9f-bc58-fca3e48b91d4.PNG)

Reply:
Yeah, I understand but I want line heights defined for heading tags in my component lib itself. Still would look into this. 
Thanks!